corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -103,6 +123,10 @@ def hash_doc_ids_python(doc_ids):
     def hash_doc_id_python(doc_id):
         if isinstance(doc_id, unicode):
             doc_id = doc_id.encode('utf-8')
+        elif isinstance(doc_id, UUID):
+            # Hash the 16-byte string
+            doc_id = doc_id.bytes
```

Comment:
it seemed like the most consistent thing to do is to just hash the actual bytestring for uuids rather than try and do it to a string representation of it. the same approach is used in `hash_doc_uuid_sql` (@snopoke fyi)